Декоративные элементы на главной странице

kozerozhechek opened the pull request:
Вставляет декоративные элементы,
иконки,
состояния.

---
:mortar_board: [Декоративные элементы на главной странице](https://up.htmlacademy.ru/htmlcss/33/user/1540215/tasks/10)

:boom: https://htmlacademy-htmlcss.github.io/1540215-sedona-33/9/

kozerozhechek committed:
Декоративные элементы

Вставляет декоративные элементы,
иконки,
состояния.

ValeryKrukovich commented on styles/slyles.css:
```diff
@@ -146,12 +154,95 @@ body {
+}
+
+.favourite:after {
+  content: "12";
```
Такое решение в корне не правильно. Ведь это число - это динамический контент. Число будет меняться постоянно. А захаркодив значие в css, ты не сможешь его поменять динамически

kozerozhechek committed:
Правки к "Декоративные элементы"

Исправила иконку с сердечком (кружок сделала отдельным элементом),
исправила меню и лого (по консультации),
Исправила сосотяние фокус (рамка) у кнопок,
изменила блок пагинации.
